Translations/Translations.csv

```diff
@@ -104,4 +104,14 @@
 "youtube_removed","Removed notifier for `{0}`","Notificaties voor `{0}` werden verwijderd"
 "youtube_no_exists","There is no notifier for `{0}`","Er zijn momenteel geen notificaties voor `{0}`"
 "lists_page_count","({0} of {1})","({0} van {1})"
-"message","Message:","Bericht:"
\ No newline at end of file
+"message","Message:","Bericht:"
+"poll_no_questionmark","No questionmark in your input, sorry.","Er ontbreekt een vraagteken in je invoer."
+"poll_no_timeout","No timeout provided.","Je hebt geen timeout (verlooptijd) voorzien."
+"poll_max_timeout","Timeout should not be longer than 60 minutes","De timeout (verlooptijd) mag niet langer zijn dan 60 minuten"
+"poll_start","<@{0}> is starting a poll with the following question:\n\n**{1}?**\n","<@{0}> is een poll begonnen met de volgende vraag:\n\n**{1}?**\n"
+"poll_yes_no","{0} - Yes\n{1} - No","{0} - Ja\n{1} - Nee"
+"poll_results","{0} **The Poll of  <@{1}> has ended**\n*{2}?*\n\n","{0} **De Poll van <@{1}> is afgelopen**\n*{2}*?\n\n"
+"poll_votes","{0} - {1}\n","{0} - {1}\n"
+"poll_usage","<question>?<timeout(minutes)> [answer 1];[answer 2];...;[answer 9]","<vraag>?<timeout (minuten)> [antwoord 1];[antwoord 2];...;[antwoord 9]"
+"poll_brief","Put a poll out for people to vote","Maak een peiling waarin mensen kunnen stemmen"
+"poll_help","Start a poll, with minimal arguments this will be a yes/no poll, if you provide options seperated by semicolons you can add up to 9 options.","Start een peiling, met minimale opties wordt dit een Ja/Nee peiling die <timeout> minuten duurt. Indien je zelf opties meegeeft door puntkomma gescheiden kunnen dat er maximaal 9 zijn."
```

Comment:
The dutch translations use the word poll everywhere except for poll_help, where it says 'peiling'.

Reply:
Ah yes, will replace with peiling to make it consistent